pydis_site/templates/home/timeline.html

```diff
@@ -12,6 +12,333 @@
+          <i class="fa fa-question"></i>
+        </div>
+
+        <div class="cd-timeline__content text-component">
+          <h2>Trivia Night</h2>
+          <p class="color-contrast-medium">
+            How well do you know Python inside out? Members got to find out in a Trivia Night event.
+            Contestants were given questions about Python's internals, its development, and more.
+            To win, contestants had to get the most questions right while being fast to answer.
+          </p>
+          <div class="flex justify-between items-center">
+            <span class="cd-timeline__date">Feb 12th, 2022</span>
+          </div>
+        </div>
+      </div>
+
+      <div class="cd-timeline__block">
+        <div class="cd-timeline__img cd-timeline__img--picture">
+          <img src="{% static "images/timeline/cd-icon-picture.svg" %}" alt="Picture">
+        </div>
+
+        <div class="cd-timeline__content text-component">
+          <h2>Creation of Events Team</h2>
+          <p class="color-contrast-medium">
+            We form the Events Team to organise and run future events. Led by Kat and comprised by
+            staff members, the goal of the team is to ultimately host more events in a more
+            sustainable way.
+          </p>
+          <div class="flex justify-between items-center">
+            <span class="cd-timeline__date">Feb 9th, 2022</span>
+          </div>
+        </div>
+      </div>
+
+
+      <div class="cd-timeline__block">
+        <div class="cd-timeline__img pastel-red cd-timeline__img--picture">
+          <i class="fa fa-code"></i>
+        </div>
+
+        <div class="cd-timeline__content text-component">
+          <h2>Deployment of Smarter Resources</h2>
+          <p class="color-contrast-medium">
+            We gave our resources pages some much needed love and
+            <a href="https://www.pythondiscord.com/resources/">
+              reorganised them into a single page
+            </a>
+            , complete with a shiny new resource filter that allows you to more quickly find
+            ones that relate to your interests, experience, learning style, and ability to pay!
+          </p>
+          <div class="flex justify-between items-center">
+            <span class="cd-timeline__date">Feb 2nd, 2022</span>
+          </div>
+        </div>
+      </div>
+
+      <div class="cd-timeline__block">
+        <div class="cd-timeline__img pastel-dark-blue cd-timeline__img--picture">
+          <i class="fa fa-users"></i>
+        </div>
+
+        <div class="cd-timeline__content text-component">
+          <h2>We hit 300 000 members!</h2>
+          <p class="color-contrast-medium">
+            Thanks to an increasing growth rate, Python Discord's membership count doubled from
+            150,000 to 300,000 in less than a year!
+          </p>
+          <div class="flex justify-between items-center">
+            <span class="cd-timeline__date">Jan ??, 2022</span>
```

Comment:
I remember looking at the public stats, but I don't think I was able to discern the specific day we reached this milestone. Some help would be appreciated :)

Reply:
19th according to grafana 
![image](https://github.com/python-discord/site/assets/22353562/ca862a36-b1aa-477f-bc64-cf7049edf0f1)

Reply:
Thanks for checking @wookie184!